_includes/layouts/base.njk

```diff
@@ -5,11 +5,28 @@
 
 	<body>
 
-		<header>
-			<em>
-				<a href="{{ '/' | url }}">{{ metadata.title }}</a>
-			</em>
-			{% include "components/nav.njk" %}
+		<header role="banner" class="header-main" id="main-navigation">
+			<div class="container container--wide container--flex">
+				<div class="header-main__logo">
+					<a href="{{ '/' | url }}">
+						<img src="/static/img/nlc-full-logoX600.png" alt="{{ metadata.title }}" />
+					</a>
+				</div>
+				<div class="header-main__nav">
+					{% include "components/nav.njk" %}
+				</div>
+				<button class="header-main__menu js-closed" aria-expanded="false" href="#main-navigation">
+					<a class="css-nav css-nav--open" href="#main-navigation" aria-label="Open menu without javascript">Open</a>
+					<a class="css-nav css-nav--close" href="#" aria-label="Close menu without javascript">Close</a>
+					<div class="header-main__menu__text closed-text" aria-label="Open menu">Open</div>
```

Comment:
Great, one small thing, can we keep the open text to say "Menu" for users with sight? I think that's more common. "Close" is fine

Reply:
Ah yes, that was a mistake. Amended